tf-training/tf-job-operator/base/application.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: app.k8s.io/v1beta1
```

Comment:
Hi Richard

Could you create tf-training/tf-job-operator/overlays/application/{kustomization.yaml, application.yaml}. You can look at jupyter/notebook-controller/overlays/application